app/assets/stylesheets/application.css.scss

```diff
@@ -25,6 +25,7 @@
 
  body {
     background-color: $snow;
+    min-width: 1500px;
```

Comment:
Hmm, let's bump this down to `1200px`? https://css-tricks.com/snippets/css/media-queries-for-standard-devices/